app/components/publishers/vacancies_component/vacancies_component.html.slim

```diff
@@ -69,8 +69,10 @@
                     = tag.div(card.labelled_item(t("jobs.publication_date"), vacancy.publish_on))
                     = tag.div(card.labelled_item(t("jobs.manage.closing_date"), vacancy.application_deadline))
                   - when "expired"
-                    - if vacancy.enable_job_applications && !vacancy_expired_over_a_year_ago?(vacancy)
+                    - if vacancy.enable_job_applications && !vacancy_expired_over_a_year_ago?(vacancy) && vacancy.job_applications.where(status: %w[submitted reviewed shortlisted unsuccessful]).count.positive?
```

Comment:
how about a simpler:
`vacancy.job_applications.where.not(status: %w[draft withdrawn]).any?`